### DISCUSSION TITLE
Graphics cannot be displayed properly in built-in terminals such as Clion

### DISCUSSION THREAD

**Discussion by QingyaoLin** [starship/starship#3164] [Q&A]
Today, I installed Starship on Windows with full expectation. When THE simple configuration was completed, I opened Terminal and saw a different terminal style, which made me feel very happy, but the pattern could not be displayed properly in Clion software. Hopefully someone can provide a solution. Thank you very much!

in terminal：
![image](https://user-images.githubusercontent.com/83944508/137676680-062c4c72-bfff-44a8-ae25-a8f7ebf9007e.png)

in clion：
![image](https://user-images.githubusercontent.com/83944508/137676734-a141d4b3-9624-4b26-9585-6e6b36568e72.png)

**Comment by chipbuster** [comment]
Unfortunately I can't test this myself because JetBrains won't accept my application for student license (dang) but you probably need to set your [fallback font](https://www.jetbrains.com/help/clion/settings-editor-font.html#typography-settings) to something that supports emoji, e.g. Noto Sans Emoji.
